### PR TITLE
adding notes from community forum

### PR DESCRIPTION
**Pull request by aleesteele**
Hi folks! These are the archived notes from February, April, and July - reformatted slightly for clarity. Please let me know if I forgot anything.

Community forum notes:
- https://annuel2.framapad.org/p/TTW-community-call-15Feb
- https://annuel2.framapad.org/p/TTW-Forum-23April
- https://annuel2.framapad.org/p/turing-way-forum-July-2024

### CODE REVIEW

**Review by Arielle-Bennett — CHANGES_REQUESTED** [review]
July notes are missing and several lines are incomplete so need reviewing against the recordings - also not clear if we made a call on sharing personal sign in data on GitHub - there was a convo with Data Protection but I can't remember what the outcome of that was?

File: governance/community-calls/20240215-community-forum.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+* Q\&A from the Community - Governance and general 
+* 5th Year Birthday celebration remarks by Malvika: [https://docs.google.com/presentation/d/1QOS\_6F7akdc0sBCJBEeJcnVQX9oOP\_6j/edit#slide=id.g2a5804106b1\_0\_0](https://docs.google.com/presentation/d/1QOS\_6F7akdc0sBCJBEeJcnVQX9oOP\_6j/edit#slide=id.g2a5804106b1\_0\_0)
+    * Celebration: [https://github.com/the-turing-way/the-turing-way/issues/3518](https://github.com/the-turing-way/the-turing-way/issues/3518)
+* Closing
+
+## 👋 Icebreaker
+
+*(Getting to know each other and this Etherpad for collaborative notes)*
+
+Name / Organisation and location you are joining from / Social media (if you'd like to connect with others) /* *What's your favourite open source community (aside from The Turing Way, of course! 😉)?* *
```
[the remark this answers is not in the document]
Comment (aleesteele) [reply governance/community-calls/20240215-community-forum.md:43]: Thanks for reviewing this @Arielle-Bennett!

@KirstieJane's has said that her preference is to keep the introductions in all of our notes, as noted in #3553. After discussing this preference with Kit (who initially recommended that we drop introductions to just archive names), we are able to keep the introductions in if we note publicly that they will be archived.

File: governance/community-calls/20240215-community-forum.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+    * We communicate openly and will be discussing governance modes that work for this decentralised structure. Probably not a very strict voting system.
+* How has working in a Working Group changed the way you engage with the project?
+    * LH/Accessibility WG: I have worked on 3 Book Dashes so far. 
+    * AB/Book Dash WG: It feels empowering to be working with a group to be worknig in an ongoing way and iterating over time and you need a consistent group for. I'm excited to improve the format
+* The Turing Way can become an example of demonstrating how we think things should be. Have you thought or planned for what support or policy can help with this? Have you thought about how the working groups can help with this process?
+    * LH/Accessibility WG: I invite anyone to participate in teh WG. Even if you think you don't know, you do. That learning can help everyone. I can hope that tinfrastructure WG that they could ask us, and I think we will try to make community aware to get guide in writing to make the community aware of what aspects are relevant for them.
```
[the remark this answers is not in the document]
Comment (aleesteele) [reply governance/community-calls/20240215-community-forum.md]: Updated - closing this comment

File: governance/community-calls/20240712-community-forum.md [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+* 16:10 Other info to share - Reminder to apply to Book Dash (Winny) (5 min)
+* 16:15 Open discussion using notes and Qs from attendees - 
+* Closing (end of 90 mins)
+
+## 👋 Icebreaker
+
+*(Getting to know each other and this Etherpad for collaborative notes)*
+
+Name (+pronouns) / Organisation and location you are joining from / Social media (if you'd like to connect with others) /* *In the communities you're a part of, what form of communications do you follow most closely?
+
```
[the remark this answers is not in the document]
Comment (aleesteele) [reply governance/community-calls/20240712-community-forum.md:81]: Closing as addressed above

File: governance/community-calls/20240423-community-forum.md [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+* 15:40 Break (5 mins)
+* 15:45 Q\&A from the Community - Speakers reply to the questions (10 mins)
+* 15:55 Practitioners hub update followed by Open Q\&A from attendees (15 mins) - Ale
+* 16:10 Other info to share - Reminder to apply to Book Dash (Winny) (5 min)
+* 16:15 Open discussion using notes and Qs from attendees - 
+* Closing (end of 90 mins)
+
+## 👋 Icebreaker
+
+*(Getting to know each other and this Etherpad for collaborative notes)*
```
[the remark this answers is not in the document]
Comment (aleesteele) [reply governance/community-calls/20240423-community-forum.md:47]: Closing as addressed above

**Comment by aleesteele** [comment]
Alright - edited and clarified notes, as well as updated July's notes too. Thanks so much for reviewing @Arielle-Bennett 🙏

**Comment by bsipocz** [comment]
This has the commits from #3159 mixed in, and has also conflicts with `main`, so I go ahead for a cleaning mission and remove the unrelated content.

**Comment by aleesteele** [comment]
Thank you so much @bsipocz! 🙏  I've been having issues with my pull requests recently - sorted through my commits in #3533 😆 It is much, much appreciated!

**Comment by bsipocz** [comment]
Yes, sadly this doesn't teach you to rebase, but for practising I would recommend doing some of the exercises at https://learngitbranching.js.org

**Comment by malvikasharan** [comment]
Those who have reviewed it, can we go ahead an merge it?
I have looked through and I don't think editing of the notes is needed - as we need these for archiving purpose.

**Review by malvikasharan — APPROVED** [review]

**Pull request merged**